app/styles/_colors.less

```diff
@@ -11,6 +11,7 @@
 @interface-color-gray-5: #a6a6a6;
 @interface-color-gray-6: #333333;
 @interface-color-gray-7: #777777;
+@interface-color-gray-8: #eeeeee;
```

Comment:
This list is craziness, but want to start consolidating the colors before actually changing them.